specs/0108-community-health-dashboard.md

```diff
@@ -33,13 +33,25 @@ Active users within a selected time frame are defined like this:
 
 Note that only users included in the all users counts can be active users. If a project contact person rejects a suggestion or submits a translation, it doesn't count as an active reviewer or contributor of the team.
 
-## Unreviewed suggestion lifespan
+## Average age of currently unreviewed suggestions
 
-A line chart showing the average age of the unreviewed suggestions in a particular month for a period of last 12 months.
+A line chart showing the average age of suggestions still unreviewed in a particular month, over the last 12 months.
 
-Data is available for each day, but aggregated by month in the chart.
+Data is available for each day, but aggregated by month in the chart. A tooltip showing the exact data at a given month appears when hovering over a chart.
+
+**Text for infobox**: How old are on average the suggestions currently unreviewed.
+
+## Time to review suggestions
+
+A line chart displaying two separate data points over the last 12 months:
+* *Age of reviewed suggestions*: average age of suggestions reviewed during the specific month.
+* *12-month average*: average age of suggestions reviewed in the 12 months before the specific month.
+
+In both cases, data is determined by looking at translations rejected or approved, ignoring self-approved or self-rejected translations.
+
+**Text for infobox**: The average number of days a contributor can expect to wait for their translation suggestions to be reviewed by managers and translators.
 
-A tooltip showing the exact data at a given month appears when hovering over a chart.
```

Comment:
I guess we should leave this line?

Reply:
Note: this is still in the old (updated) section, but adding it to the new one.